README.md

```diff
@@ -76,12 +74,25 @@ collections:
 - name: ansible.windows
 ```
 
+**NOTE**: For Ansible 2.9, you may not see deprecation warnings when you run your playbooks with this collection. Use this documentation to track when a module is deprecated.
```

Comment:
This collection won't work for 2.9, or at least the full collection doesn't. We are only targeting 2.10+.